internal/sourcecode/templates/laravel/standard/.dockerignore

```diff
@@ -0,0 +1,31 @@
+# excludes from the docker image/build
+
+# 1. Ignore everything (ensures don't upload .git, .env etc)
+**
```

Comment:
Interesting! Never seen this technique used before.

Reply:
Same here! I have a starting point from Greg's work (Greg Sanderson is a guest in the `#fly-content` channel). This seems pretty reasonable for a Laravel app to me.